.gitignore

```diff
@@ -127,3 +127,17 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+tests/scripts/data
+tests/data
+pv-solar-farm-forecasting.code-workspace
+.vscode
+.gitattributes
+conftest.py
+.flake8
+.pre-commit-config.yaml
+environment.yml
```

Comment:
I wouldn't add all these to the gitignore, as we'll need a lot of these files in the end.

Reply:
Yeah, sure, I moved all of them to a different PR as suggested, I just put them in the current PR `.gitignore`, so as not to have same files in every PR! I'll remove them  from `.gitignore` when merging with the main branch